shared/js/background/components/dashboard-messaging.js

```diff
@@ -0,0 +1,92 @@
+import browser from 'webextension-polyfill';
+import { breakageReportForTab, getDisclosureDetails } from '../broken-site-report';
+import { dashboardDataFromTab } from '../classes/privacy-dashboard-data';
+import { registerMessageHandler } from '../message-handlers';
+import { getCurrentTab } from '../utils';
+import { isFireButtonEnabled } from './fire-button';
+
+/**
+ * Message handlers for communication from the dashboard to the extension background.
+ *
+ * Note, additional message handles for toggle reports is in the separate ToggleReports component.
```

Comment:
I think there are other dashboard messages that we didn't move here? E.g. for the fire button. Maybe we should mention some of those in the comment here, or move the handlers into this component too? I think the docs here might help https://duckduckgo.github.io/privacy-dashboard/modules/Browser_Extensions_integration.html

Reply:
I've updated that docstring to include all messages and where they are being handled. Thanks for pointing to those docs, super helpful!

Reply:
Ace, looks good. Actually let's include that link in the docstring too for folks?